qt/aqt/fields.py

```diff
@@ -41,28 +42,53 @@ def __init__(
         self.model = nt
         self.mm._remove_from_cache(self.model["id"])
         self.change_tracker = ChangeTracker(self.mw)
-        self.form = aqt.forms.fields.Ui_Dialog()
-        self.form.setupUi(self)
+
         self.setWindowTitle(
             without_unicode_isolation(tr.fields_fields_for(val=self.model["name"]))
         )
-        disable_help_button(self)
-        self.form.buttonBox.button(QDialogButtonBox.StandardButton.Help).setAutoDefault(
-            False
-        )
-        self.form.buttonBox.button(
-            QDialogButtonBox.StandardButton.Cancel
-        ).setAutoDefault(False)
-        self.form.buttonBox.button(QDialogButtonBox.StandardButton.Save).setAutoDefault(
-            False
-        )
-        self.currentIdx: Optional[int] = None
-        self.fillFields()
-        self.setupSignals()
-        self.form.fieldList.setDragDropMode(QAbstractItemView.DragDropMode.InternalMove)
-        self.form.fieldList.dropEvent = self.onDrop  # type: ignore[assignment]
-        self.form.fieldList.setCurrentRow(open_at)
-        self.exec()
+
+        if os.getenv("ANKI_EXPERIMENTAL_FIELDS_WEB"):
+            form = aqt.forms.fields_web.Ui_Dialog()
+            form.setupUi(self)
+
+            self.webview = form.webview
+            self.webview.set_title("fields")
+
+            self.show()
+            self.refresh()
+            self.webview.set_bridge_command(self._on_bridge_cmd, self)
+            self.activateWindow()
+
+        else:
+            self.form = aqt.forms.fields.Ui_Dialog()
```

Comment:
Minor nit: if you return inside the if clause, then the original code doesn't need to be indented, and it will result in a smaller diff/less chance of a conflict.